<a id="Description:"><a/>
<a id="Data Availability:"><a/>
<a id="Key Features:"><a/>
<a id="Bioinformatics Drawing:"><a/>
<a id="Software Repositories used in this project:"><a/>
<a id="Funding Sources:"><a/>

# Pangenomic analyses of antibiotic-resistant *Campylobacter jejuni* reveal unique lineage distributions and epidemiological associations
***Campylobacter jejuni* Bioinformatics Analysis - Michigan Department of Health and Human Services (MDHHS) and Manning Lab Collaboration**

## Description:
This repository contains the scripts utilized for bioinformatics and statistical analysis associated with [Rodrigues et al., 2023 publication in Microbial Genomics](https://doi.org/10.1099/mgen.0.001073).
The work is a collaboration between the MDHHS and the [Manning Lab](https://manninglab.natsci.msu.edu/) at Michigan State University.

## Data Availability: 
Sequences are available at BioProjects PRJNA305291, PRJNA368990, and PRJNA951423, and new MLSTs identified in this study have been deposited to [PubMLST](https://pubmlst.org/) with the designations of ST-12343, and ST-12344.
As described in the publication, a [Microreact Link](https://microreact.org/project/dCGmMiVxreTFjNaqKC68gX-rodrigues-ja-et-al-microbial-genomics-2023) is available to view the phylogeographic analysis.

## Key Features:
- Raw Data Management
- Genome Assembly and Annotation
- Comparative Genomics
- Phylogenetics
- Antibiotic Resistance
- Data Visualization
- Statistical Analyses

## Bioinformatics Drawing:
Please see the incredibly artistic flowchart below! This flowchart has a step-by-step breakdown of the bioinformatics pipeline, including the software used, inputs, outputs, and specific files for this publication.
![An incredible artistic cartoon drawing of the methods description found in Rodrigues et al, 2023]()


## Software Repisotires, Resources, and Tools used in this project:
- [ABRICATE](https://github.com/tseemann/abricate)
- [AMRFinderPlus](https://github.com/ncbi/amr)
- [Center for Genomic Epidemiology](https://www.genomicepidemiology.org/)
- [FastANI](https://github.com/ParBLiSS/FastANI)
- [FastQC](https://github.com/s-andrews/FastQC)
- [ITOL](https://itol.embl.de/)
- [MicroReact](https://microreact.org/)
- [MLST](https://github.com/tseemann/mlst)
- [MultiQC](https://github.com/ewels/MultiQC)
- [PubMLST](https://pubmlst.org/)
- [Prokka](https://github.com/tseemann/prokka)
- [QUAST](https://github.com/ablab/quast)
- [Roary](https://github.com/sanger-pathogens/Roary)
- [SPAdes](https://github.com/ablab/spades)
- [SpilteTree](https://uni-tuebingen.de/en/fakultaeten/mathematisch-naturwissenschaftliche-fakultaet/fachbereiche/informatik/lehrstuehle/algorithms-in-bioinformatics/software/splitstree/)
- [StarAMR](https://github.com/phac-nml/staramr)
- [Trimmomatic](http://www.usadellab.org/cms/index.php?page=trimmomatic)

## Funding Sources: 
This project was funded by the Michigan Sequencing and Academic Partnerships for Public Health Innovation and Response (MI-SAPPHIRE) initiative at the MDHHS, which was established and supported by funds through the Centers for Disease Control and Prevention (CDC). The CDC grant was entitled Epidemiology and Laboratory Capacity for Prevention and Control of Emerging Infectious Diseases Enhancing Detection Expansion programme (6NU50CK000510-02-087) as well as Michigan State University (MSU), AgBioResearch, and the MSU Foundation to the Manning Lab. Additional trainee support was provided to Jose A. Rodrigues via the Department of Microbiology and Molecular Genetics, the Ronald and Sharon Rogowski Fellowship, the Thomas S. Whittam travel award, and the MSU College of Osteopathic Medicine.


